src/reflow/mod.rs

```diff
@@ -0,0 +1,286 @@
+//! Reflow documentation comments to a desired line width.
+//!
+//! Note that for commonmark this might not be possible with links.
+//! The reflow is done based on the comments no matter the content.
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+use crate::checker::Checker;
+use crate::documentation::{CheckableChunk, Documentation};
+use crate::util::sub_chars;
+use crate::{ContentOrigin, Detector, LineColumn, Range, Span, Suggestion, SuggestionSet};
+
+use indexmap::IndexMap;
+use log::trace;
+use pulldown_cmark::{Event, Options, Parser, Tag};
+
+mod config;
+pub use config::ReflowConfig;
+
+mod iter;
+pub use iter::{Gluon, Tokeneer};
+
+#[derive(Debug)]
+pub struct Reflow;
+
+impl Checker for Reflow {
+    type Config = ReflowConfig;
+    fn check<'a, 's>(docu: &'a Documentation, config: &Self::Config) -> Result<SuggestionSet<'s>>
+    where
+        'a: 's,
+    {
+        let wrapper = textwrap::Wrapper::new(config.max_line_length)
```

Comment:
That's my old stuff right? Since it's never gonna work, maybe just remove it right away?

Reply:
That would be the next step :+1: